traffic_ops/traffic_ops_golang/riak.go

```diff
@@ -0,0 +1,204 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/apache/incubator-trafficcontrol/lib/go-log"
+	"github.com/apache/incubator-trafficcontrol/traffic_ops/tostructs"
+	"github.com/basho/riak-go-client"
+	"github.com/jmoiron/sqlx"
+	"github.com/lestrrat/go-jwx/jwk"
+	"io/ioutil"
+	"net/http"
+)
+
+const riak_port = 8087
+const cdn_uri_keys_bucket = "cdn_uri_sig_keys" // riak namespace for cdn uri signing keys.
+
+func getStringValue(resp *riak.FetchValueResponse) (string, error) {
+	var obj *riak.Object
+
+	if len(resp.Values) > 0 {
```

Comment:
You are currently requesting via key, so I _think_ it's impossible to get more than 1 value here. But it feels wrong not to check for the case anyway and error out.

Reply:
Okay, I agree, I'll change it to ==